querqy-for-lucene/querqy-solr/src/main/java/querqy/solr/QuerqyDismaxQParser.java

```diff
@@ -194,4 +175,27 @@ public SearchEngineRequestAdapter getSearchEngineRequestAdapter() {
     public List<Query> getFilterQueries() {
         return luceneQueries == null ? null : luceneQueries.filterQueries;
     }
+
+    public RankQuery getRankQuery() {
```

Comment:
Minor change: can we make this method return an Optional instead of +/- null? (this would be more expressive)